content/module-reference/processing-modules/color-equalizer.md

```diff
@@ -1,7 +1,156 @@
 ---
-title: color equalizer (placeholder)
+title: color equalizer
 id: color-equalizer
 weight: 10
 applicable-version: 4.8
+tags: color color-processing
+working-color-space: UCS
+view: darkroom
+masking: true
+include_toc: true
 ---
 
+Selectively adjust the hue, saturation, and brightness of pixels based on the color.
+
+Color Equalizer is essentially an implementation of the Color Zones module, but works in the scene referred part of darktable's pixelpipe.
+
+This module works in darktable's UCS 22 color space.
+
+The entire hue range is represented as a curve with eight overlapping sections divided by eight fixed nodes. Choose whether you wish to adjust (select) pixels based on their hue, saturation, and brightness. You can then use the equalizer interface nodes, on their respective tabs, to adjust the hue, saturation, and brightness of ranges of pixels selected via this method.
```

Comment:
You can't choose to adjust/select by saturation or brightness. All of the selections are by hue. It should perhaps say something like "Pixels are selected based on their hue, and the module can be used to adjust the hue, saturation, or brightness of those selected pixels"